apollo-router-core/src/json_ext.rs

```diff
@@ -276,7 +426,7 @@ where
 /// This can be composed of strings and numbers
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, Default)]
 #[serde(transparent)]
-pub struct Path(Vec<PathElement>);
+pub struct Path(pub Vec<PathElement>);
```

Comment:
I think we need to add `#[doc(hidden)]` on this and the `PathElement`.

Reply:
fixed in 1e5489c